### PR TITLE
DCOS-39677: Capitalize only first letter in table header

### PR DESCRIPTION
**Pull request by GeorgiSTodorov**
This `uppercase` css property is removed in order to make only first letter in headers capital.

related jira: https://jira.mesosphere.com/browse/DCOS-39677

### CODE REVIEW

**Comment by lilysamimi** [comment]
Following up on what @weblancaster commented on, I'm only seeing the titles in lowercase and not in sentence case. 

https://cl.ly/2Q2p2d3p2g0l

**Comment by GeorgiSTodorov** [comment]
@weblancaster @lilysamimi Fixed.

screenshot from 2018-08-07 11-21-23

File: packages/table/components/HeaderCell.tsx [hunk 1]
```diff
@@ -2,6 +2,6 @@ import { default as Cell } from "./Cell";
 import styled from "react-emotion";
 
 export default styled(Cell)`
-  text-transform: uppercase;
+  text-transform: capitalize;
```
Comment (DanielMSchmidt) [inline packages/table/components/HeaderCell.tsx]: Nice 👍

Comment (GeorgiSTodorov) [reply packages/table/components/HeaderCell.tsx]: It's marked as a bug, so I guess it should be a `fix`.

**Comment by DanielMSchmidt** [comment]
@weblancaster Can you take another look?

**Pull request merged**